lib/matplotlib/axes/_axes.py

```diff
@@ -816,22 +809,33 @@ def axhspan(self, ymin, ymax, xmin=0, xmax=1, **kwargs):
         0.5=middle, 1.0=right but the *y* location is in data
         coordinates.
 
-        Return value is a :class:`matplotlib.patches.Polygon`
-        instance.
-
-       Examples:
-
-        * draw a gray rectangle from *y* = 0.25-0.75 that spans the
-          horizontal extent of the axes::
-
-            >>> axhspan(0.25, 0.75, facecolor='0.5', alpha=0.5)
+        Parameters
+        ----------
+        ymin : float
+               Lower limit of the horizontal span in data units.
+        ymax : float
+               Upper limit of the horizontal span in data units.
+        xmin : float, optional, default: 0
+               Lower limit of the vertical span in axes (relative
+               0-1) units.
+        xmax : float, optional, default: 1
+               Upper limit of the vertical span in axes (relative
+               0-1) units.
 
-        Valid kwargs are :class:`~matplotlib.patches.Polygon` properties:
+        Returns
+        -------
+        Polygon : `~matplotlib.patches.Polygon`
 
-        %(Polygon)s
```

Comment:
Only the general line (i.e., `kwargs`), but not this substitution of all the keyword arguments.